cmake/configure/configure_assimp.cmake

```diff
@@ -0,0 +1,20 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2014 - 2015 by the deal2lkit authors
+##
+## This file is part of the deal2lkit library.
+##
+## The deal2lkit library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal2lkit distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# Configuration for the sundials library:
```

Comment:
Could you also add
```
MACRO(FEATURE_ASSIMP_FIND_EXTERNAL var)

	FIND_PACKAGE(ASSIMP)

		IF(ASSIMP_FOUND)

		SET(${var} TRUE)

    #
    # Set ASSIMP_DIR to something meaningful if empty
    #
    IF("${ASSIMP_DIR}" STREQUAL "")
      SET(ASSIMP_DIR "<system location>")
    ENDIF()

	ENDIF()
ENDMACRO()
```

which helps in the definition of useful variables such as `D2K_INCLUDE_DIRS` ?